src/PluginProcessor.cpp

```diff
@@ -322,6 +294,33 @@ void ProteusAudioProcessor::loadConfig(File configFile)
     this->suspendProcessing(false);
 }
 
+void ProteusAudioProcessor::LSTMProcess(const AudioBuffer<float>& buffer, dsp::AudioBlock<float> block, float driveValue)
+{
+    const auto block44k = resampler.processIn(block);
+    
+    if (buffer.getNumChannels() == 2)
+    {
```

Comment:
Allows processing both channels (L&R) concurrently, boosting performance, especially when running multiple versions of this plugin, i.e. one for drive pedal, one for amp.